libcodechecker/analyze/store_handler.py

```diff
@@ -0,0 +1,526 @@
+# -------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -------------------------------------------------------------------------
+
+from hashlib import sha256
+import json
+import os
+import tempfile
+import zlib
+
+import shared
```

Comment:
Import order is: systemwide, external, API, internal, local.

So: 
 

1.  `import zlib`
2. (empty line)
3. `import sqlalchemy`
4. (empty line)
5. `import shared`
6. `from codeCheckerDBAcess...`